oeo.omn

```diff
@@ -2586,6 +2586,15 @@ Class: GreenhouseGas
         Pollutant
     
     
+Class: Grid
+
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000115> "A grid is an assembly of systematically connected elements that can work as a supply system."
```

Comment:
```suggestion
        <http://purl.obolibrary.org/obo/IAO_0000115> "A grid is an assembly of systematically connected artificial objects (object aggregate) that can work as a supply system."
```
I'd like to include the parent class (object aggregate) in the def, see Aristotelian definitions in the wiki. It's not a beauty, though. What do other reviewers think?

Reply:
maybe use "object aggregate" instead of "assembly" at the beginning? To clearly indicate the subclass relationship. I like the "artificial objects" change